### ISSUE TITLE
Ecowitt WN32B not detected / decoded

### ISSUE THREAD

**Issue by antamy** [merbanan/rtl_433#2529]
I'm new to RTL_433, so apologies upfront if I'm not conveying the correct information.  This is similar to issue #2303, but seems different enough that I'm creating a new issue.

The device is an Ecowitt WN32B 915MHz (no barometric pressure).  I applied the code changes in #2303, but they don't have any effect. 

I do see this in the output:

```
rtl_433 -f 915.05M
...
codes     : {170}d5aaaaaaaaaaaa2dd4d122553cffff825c000000000
```

which contains the correct data using the decode info in `fineoffset.c` 

```
aa 2d d4 d1 22 66 3a ff ff 91 5c
         MI IT TT HH PP PP CC XX
```
Running with `-vv` doesn't show show `fineoffset_WH25_callback` anywhere, so I assume the code in `fineoffset.c` is not being invoked.  I'm not sure what / where to change to enable this device to be detected.

I have also tried running `rtl_433 -f 915.05M -vv -S unknown` .  I clearly see the `{170}` codes similar to above in the output, roughly every minute, however when I analyze the log files, those codes don't show up.  For example, I get this and analyze the output in the g072 file and don't see the data (same happens if I use `-S all`) :

```
*** Saving signal to file g071_915.05M_1000k.cu8 (35926 samples, 131072 bytes)
[fineoffset_WH0290_callback] Checksum error: 7a 02
[ambientweather_whx_decode] WH31E/WH31B/WH40 detected, buffer is 166 bits length
[ambientweather_whx_decode] unknown message type d1 (expected 0x30/0x40/0x68)
[fineoffset_WH51_callback] short package. Header index: 72
codes     : {170}d5aaaaaaaaaaaa2dd4d122543cffff815c000000000
[fineoffset_wh1080_callback] Fine Offset WH1080 data
codes     : {80}ffd122543cffff815c00
[lacrosse_breezepro_decode] Wrong packet length: 170
[lacrosse_wr1_decode] Packet too long: 170 bits
[lacrosse_th_decode] packet length: 170
[lacrosse_th_decode] Sync word not found
[lacrosse_r1_decode] Packet too short: 170 bits
[honeywell_cm921_decode]
codes     : {344}f999e666666666666666666666660679f99879818606199807f87ffffffff000ccfc000000000000000000
[honeywell_cm921_decode] preamble_start=344 start=374 len=-30
[fineoffset_wh31l_decode] WH31L detected, buffer is 151 bits length
[lacrosse_tx34_callback] LaCrosse IT frame bad CRC: calculated 9b, received ff
[fineoffset_wn34_decode]
codes     : {72}d122543cffff815c00
[fineoffset_WH0290_callback] Checksum error: 7a 02
[ambientweather_whx_decode] WH31E/WH31B/WH40 detected, buffer is 166 bits length
[ambientweather_whx_decode] unknown message type d1 (expected 0x30/0x40/0x68)
[fineoffset_WH51_callback] short package. Header index: 72
codes     : {170}d5aaaaaaaaaaaa2dd4d122543cffff815c000000000
[fineoffset_wh1080_callback] Fine Offset WH1080 data
codes     : {80}ffd122543cffff815c00
[lacrosse_breezepro_decode] Wrong packet length: 170
[lacrosse_wr1_decode] Packet too long: 170 bits
[lacrosse_th_decode] packet length: 170
[lacrosse_th_decode] Sync word not found
[lacrosse_r1_decode] Packet too short: 170 bits
[honeywell_cm921_decode]
codes     : {343}f333cccccccccccccccccccccccc0cf3f330f3030c0c33300ff0ffffffffe00199f8000000000000000000
[honeywell_cm921_decode] preamble_start=343 start=373 len=-30
[fineoffset_wh31l_decode] WH31L detected, buffer is 151 bits length
[lacrosse_tx34_callback] LaCrosse IT frame bad CRC: calculated 9b, received ff
[fineoffset_wn34_decode]
codes     : {72}d122543cffff815c00
[lacrosse_tx31u_decode] LaCrosse TX31U-IT detected
[lacrosse_tx31u_decode] LaCrosse TX31U-IT bad CRC: calculated 7a, received 81
*** Saving signal to file g072_915.05M_1000k.cu8 (86643 samples, 262144 bytes)
```

Any ideas on how I can proceed?

Thanks

[g072_915.05M_1000k.zip](https://github.com/merbanan/rtl_433/files/11746571/g072_915.05M_1000k.zip)

**Comment by ProfBoc75** [comment]
hi @antamy 

With the changes from #2303 it works with your sample. 

```
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
time      : @0.060813s
model     : Fineoffset-WH32                        ID        : 18
Battery   : 1            Temperature: 19.6 C       Humidity  : 60 %          Integrity : CRC
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
time      : @0.104606s
model     : Fineoffset-WH32                        ID        : 18
Battery   : 1            Temperature: 19.6 C       Humidity  : 60 %          Integrity : CRC
```

I guess your file fineoffset.c is not properly updated

**Comment by antamy** [comment]
> I guess your file fineoffset.c is not properly updated

Thanks for looking at this.  I haven't done any programming for years, and getting my head round all the tools and environment is still a work in progress.  My issue was that the `make install` actually didn't work (it gave an error on updating man) and I assumed that the man piece could be ignored.  Turns out that the other steps never ran, so I was using an older version of rtl_433.  Running `sudo make install` fixed my issue.

That said, if I hadn't encountered the problem, I wouldn't have learned as much about rtl_433 as I did last night trying to figure it out!
